fixtures/expectations/topology_depth_1_child_tree.json

```diff
@@ -1,33 +1,53 @@
-[
-  {
-    "id": "018681fd-5770-336f-227c-259435d7fc6b",
-    "external_id": "dummy/logistics-api",
-    "parent_id": "018681fc-e54f-bd4f-42be-068a9a69eeb5",
-    "name": "logistics-api",
-    "labels": {
-      "telemetry": "enabled"
-    },
-    "status": "warning",
-    "type": "Application",
-    "owner": "logistics-team",
-    "configs": null,
-    "path": "018681fc-e54f-bd4f-42be-068a9a69eeb5",
-    "summary": {
+{
+  "healthStatuses": [
+    "healthy",
+    "unhealthy"
+  ],
+  "tags": [
+    {
+      "telemetry": [
+        "enabled"
+      ]
+    }
+  ],
```

Comment:
maybe tags could just be an object instead of an array with objects, if order doesn't matter here. Not sure how this will be structured if there were multiple tags, but I think this wouldn't be an ideal way to access the tags from the frontend.

So, instead of 
```json
{
  "tags": [
    {
      "telemetry": ["enabled"]
    },
    {
      "security": ["critical", "high"]
    }
  ]
}
```

we could do this
```json
{
  "tags": 
    {
      "telemetry": ["enabled"],
      "security": ["critical", "high"]
    }
}
```

Reply:
Yeah, makes sense

👍🏽